### ISSUE TITLE
Folder Alias not properly supported on macOS

### ISSUE THREAD

**Issue by canoben** [ajeetdsouza/zoxide#855]
CDing to an alias file in macOS correctly changes the current directory to the folder pointed to by the alias file.

zoxide only sees a file and emits an error.

**Comment by ajeetdsouza** [comment]
What is an alias file - do you mean a symbolic link? What error is zoxide emitting?

**Comment by canoben** [comment]
Tried it today, still not working.

**Comment by ajeetdsouza** [comment]
> What is an alias file - do you mean a symbolic link? What error is zoxide emitting?

Can you answer this?

**Comment by canoben** [comment]
Yes, it's like a symbolic link. zoxide simply emits a "zoxide: no match found" message.
`╭─ ~/desktop/test ───────────────────────────────────────────────────────────22:40:20
╰─ ls -al
   rwxr-xr-x   5   charel   wheel    160 B     Fri Jan  3 22:39:41 2025    ./
   rwx------   8   charel   staff    256 B     Fri Jan  3 22:40:13 2025    ../
   rw-r--r--   1   charel   wheel      6 KiB   Fri Jan  3 22:40:13 2025    .DS_Store 
   rw-r--r--   1   charel   wheel    872 B     Fri Feb 26 18:41:01 2021    Ranger 
   rw-r--r--   1   charel   staff      4 B     Fri Jan  3 22:39:39 2025    test.txt 

╭~/desktop/test ────────────────────────────────────────────────────────────22:40:23 
╰─ cd Ranger
zoxide: no match found`

**Comment by ajeetdsouza** [comment]
What do you mean by "like a symbolic link"? How have you created it?

**Comment by canoben** [comment]
By using the macOS Finder command "Create Alias".
